### PR TITLE
[BACK-2679] Resolve dependabot issues

### PR DESCRIPTION
**Pull request by darinkrauss**
- Resolve dependabot issues
- Update to node v20.11.0
- Remove request dependency
- Add lint via jshint
- Add .nvmrc
- Update .travis.yml
- Update Dockerfile
- Bump version
- Remove extraneous yarn.lock
- https://tidepool.atlassian.net/browse/BACK-2679

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v20.11.0
```
Comment (jh-bate) [inline .nvmrc:1]: `v20.10.0` ?

File: .travis.yml [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
 
-install:
-  - npm install
+services:
+  - docker
 
 env:
   global:
-  - MONGODB=5.3.2
+    - MONGODB=mongodb-linux-x86_64-ubuntu2204-6.0.12
```
Comment (jh-bate) [inline .travis.yml:13]: I assume this doesn't matter but not 6.0.11 stated??

**Review by jh-bate — APPROVED** [review]
couple of version comments

File: .nvmrc (hunk 1 above)
Comment (darinkrauss) [reply .nvmrc:1]: Updated PR description.

**Review by lostlevels — APPROVED** [review]

**Pull request merged**